drivers/display/display_ili9340_seeed_tftv2.c

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright (c) 2018 - 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <display.h>
+#include "display_ili9340.h"
+#include "display_ili9340_rgb565.h"
+#include "display_ili9340_rgb888.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(display_ili9340_seeed_tftv2, CONFIG_DISPLAY_LOG_LEVEL);
+
+/*
+ * Derive from Seeed 2.8 inch TFT Touch Shield v2.0 sample code.
+ *
+ * https://github.com/Seeed-Studio/TFT_Touch_Shield_V2
+ */
+
+/**@brief Function to wait milliseconds.
+ */
+static void wait_ms(u16_t t)
```

Comment:
Use k_sleep directly instead of wrapping it in an extra function.

Reply:
Will do.